xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-lightbox/xwiki-platform-image-lightbox-ui/src/main/resources/XWiki/Lightbox/LightboxTranslations.xml

```diff
@@ -57,7 +57,8 @@ image.lightbox.fullscreen.title=Fullscreen
 image.lightbox.gallery.title=Image gallery
 image.lightbox.nextSlide.title=Next image
 image.lightbox.prevSlide.title=Previous image
-image.lightbox.publisher.label=Image publisher</content>
+image.lightbox.publisher.label=Image publisher
+image.lightbox.toggler.label=Open the image options</content>
```

Comment:
Shouldn't this rather be "Open the image lightbox"?

Reply:
I'm not sure it's clear for a regular user what a lightbox is.
For the sake of avoiding technical wording, I thought 'image options' was good enough of a description for the lightbox.
I'm not sure the word 'Lightbox' is displayed anywhere in the UI for regular users (there's an admin section using it though). So if there's no explanation, the word Lightbox doesn't mean anything to a regular user.

Reply:
> What about "Open the image actions"?  I think I thought this for opening the lightbox, but this only opens the small popover with the option to open the lightbox, so I think this is fine in the end, regardless if we choose "options" or "actions".

:+1: Addressed in https://github.com/xwiki/xwiki-platform/pull/2281/commits/4c3508002ae34a28b98eaa03a62b033c7bd7ef23

> Also, I wonder if we really need the "open the" prefix.

I don't really have a precise and trustable reference in mind, but I think it's a best practice to use action verbs to label buttons. A button can do a bunch of things, so it's better to highlight the specific action.
There is [this article](https://uxmovement.medium.com/5-rules-for-choosing-the-right-words-on-button-labels-dc3f74c2c2a3) on Medium highlighting it as Rule 1. , but I don't know the author(s) and can't be sure it's a trustworthy source.